homeassistant/components/ziggo_mediabox_xl/media_player.py

```diff
@@ -43,18 +48,22 @@
 )
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+def setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
     """Set up the Ziggo Mediabox XL platform."""
 
     hass.data[DATA_KNOWN_DEVICES] = known_devices = set()
 
     # Is this a manual configuration?
-    if config.get(CONF_HOST) is not None:
-        host = config.get(CONF_HOST)
+    if (host := config.get(CONF_HOST)) is not None:
         name = config.get(CONF_NAME)
         manual_config = True
     elif discovery_info is not None:
-        host = discovery_info.get("host")
+        host = discovery_info["host"]
```

Comment:
This change is because `socket.gethostbyname` doesn't allow `None` as parameter, so we might as well exit early.